### ISSUE TITLE
Cannot open .geo file through the windows file association

### ISSUE THREAD

**Issue by MrClock8163** [zsiki/GeoEasy#188]
When trying to open a .geo file directly from the windows file explorer, GeoEasy cannot open the file if the path contains any whitespaces. The file path received from the shell seems to be split up at every whitespace character, and the programs attempts to load each part individually. This is inconsistent with the regular **Load** function of the **File** menu, where such spaces cause no issues.

Eg. path `D:\Test_folder\folder with space\testgeo.geo`

Through the file association:
![image](https://user-images.githubusercontent.com/48679831/225068809-96d91843-2387-44f3-9630-beec0892c0cf.png)
![image](https://user-images.githubusercontent.com/48679831/225068866-38e990f4-cedf-476e-a012-4b0c9115aa1d.png)
![image](https://user-images.githubusercontent.com/48679831/225068901-4a532a9f-b067-477d-89cf-0be2d0e0a0d9.png)

Through the File menu:
![image](https://user-images.githubusercontent.com/48679831/225069513-7216e56b-6f51-489f-8f8b-b5b320cad1fa.png)

OS: Windows 10
GeoEasy version: 3.2.1

I would speculate that the issue may be caused by the ability to load multiple files at once through the file browser from the File menu of the main window.

**Comment by zsiki** [comment]
New binary development version is available on http://digikom.hu/english/geo_easy_e.html.
Please test it.

**Comment by MrClock8163** [comment]
I tested the new version, and the file association works without issue.
The only inconsistency now is that the log window does not show the file was opened.
But I guess this is not really a big deal.
![image](https://user-images.githubusercontent.com/48679831/225444438-9bde00e4-648a-44e8-a210-b0bd349354e7.png)

I ran into another bug though, concerning the saving of parameters. Should I open a new issue for that?
